elastic/kibana
# [ResponseOps] Hide the alert delay in the rule flyout

## Summary

Updates the ui to hide the alert delay under an accordion.

<img width="625" alt="Screen Shot 2024-02-15 at 1 37 40 PM" src="https://github.com/elastic/kibana/assets/109488926/d8f05b00-3732-492b-9d64-3a31331597bc">

<img width="613" alt="Screen Shot 2024-02-15 at 1 38 49 PM" src="https://github.com/elastic/kibana/assets/109488926/7d2d0e67-e24e-481a-8541-68ae2cf09a19">

### Checklist


### To verify

- Create a new rule
- Verify that the alert delay is hidden
- Click the accordion toggle and set the alert delay
- Save the rule
- Edit the rule and verify that the alert delay is hidden. When you click the accordion toggle verify that alert delay is correct
